docs/src/main/asciidoc/container-image.adoc

```diff
@@ -0,0 +1,136 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Container Images
+
+include::./attributes.adoc[]
+
+Quarkus provides extensions for building (and pushing) container images. Currently it supports:
+
+* jib
+* docker
+* s2i
+
+== Building
+
+To build a container image for your project, you just need to specify `quarkus.container-image.execution=build` either to the `application.properties` or as a system property.
+
+[source, subs=attributes+]
+----
+mvn clean package -Dquarkus.container-image.exection=build
```

Comment:
before this command will do anything I would need to at least add one of the extensions found below, right? maybe swap the sections or at least point out you shouldn;t run that command without first adding the deeondency.

Reply:
Swapped sections.